python/src/HMatrixImplementation_doc.i.in

```diff
@@ -175,7 +175,16 @@ OT_HMatrix_assembleTensor_doc
 Parameters
 ----------
 method : str
-    Factorization method, either one of: LDLt, LLt or LU"
+    Factorization method, either one of: LDLt, LLt or LU
+
+Notes
+-----
+The factorization embeds an automatic regularization procedure based
+on an approximation of the largest eigenvalue module. Its computation
+is done using a power iteration, controlled by the
+'HMatrix-LargestEigenValueRelatveError' and
+'HMatrix-LargestEigenValueIterations keys in the :class:`~openturns.ResourceMap`.
```

Comment:
'HMatrix-LargestEigenValueIterations' --> missing '

Reply:
Here, `ResourceMap.GetAsUnsignedInteger("HMatrix-LargestEigenValueIterations")` returns 10. How do you get this error?

Reply:
no it is a typo. There is a missing "'" (after `LargestEigenValueIterations` word)